app/controllers/users_controller.rb

```diff
@@ -11,7 +11,13 @@ def new
   def index
     respond_to do |format|
       format.html
-      format.json { render json: UsersDatatable.new(view_context, current_ability) }
+      format.json {
+        @users = User.includes(:associate_consultant,
+                               {:associate_consultant => :reviewing_group},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.